### PR TITLE
[Addon] Upgrade addon to use WasmEdge C API

### PR DESCRIPTION
**Pull request by dm4**
- Upgrade WasmEdge to 0.8.1
- Use WasmEdge C API
- Use `wasmedge/wasmedge` container image for CI
- Install prebuilt WasmEdge library instead of building from source in CI

Something about tests:
- Because [rustwasmc][] still use `require('ssvm')`, here I use `sed -i "s/require('ssvm')/require('wasmedge-core')/"` to fix tests before [rustwasmc][] upgrade.
- Because test [rsa-example][] no longer be able to build now, I remove it from our test cases'

[rustwasmc]: https://github.com/second-state/rustwasmc
[rsa-example]: https://github.com/second-state/wasm-learning/tree/master/nodejs/rsa_example

### CODE REVIEW

File: .github/workflows/external-test.yml [hunk 1]
```diff
@@ -31,65 +31,68 @@ jobs:
       with:
         repository: second-state/wasm-learning
 
+    - name: Install WasmEdge
+      run: |
+        wget https://github.com/second-state/WasmEdge-go/releases/download/v0.8.1/install_wasmedge.sh
```
Comment (hydai) [inline .github/workflows/external-test.yml]: Maybe we should move this script into our `preinstall.sh`

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -11,28 +11,42 @@ on:
...
 
     steps:
     - uses: actions/checkout@v2
-      with:
-        submodules: recursive
 
-    - name: Install dependencies
+    - name: Install WasmEdge
+      run: |
+        wget https://github.com/second-state/WasmEdge-go/releases/download/v0.8.1/install_wasmedge.sh
```
Comment (hydai) [inline .github/workflows/main.yml]: Ditto.

File: binding.gyp [hunk 3]
```diff
@@ -7,69 +7,20 @@
...
-        "ssvm-core/thirdparty/easyloggingpp/easylogging++.cc",
       ],
       "include_dirs": [
         "<!@(node -p \"require('node-addon-api').include\")",
         "src",
-        "ssvm-core/include",
-        "ssvm-core/thirdparty",
-        "ssvm-core/thirdparty/googletest/include",
+        "/usr/local/include",
         "/usr/lib/llvm-10/include",
```
Comment (hydai) [inline binding.gyp]: This line should be removed too.

File: package.json [hunk 4]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "ssvm",
+  "name": "wasmedge-core",
   "version": "0.7.3",
```
Comment (hydai) [inline package.json]: `0.8.1`

**Review by hydai — CHANGES_REQUESTED** [review]

File: .github/workflows/external-test.yml (hunk 1 above)
Comment (dm4) [reply .github/workflows/external-test.yml]: done

File: .github/workflows/main.yml (hunk 2 above)
Comment (dm4) [reply .github/workflows/main.yml]: done

File: binding.gyp (hunk 3 above)
Comment (dm4) [reply binding.gyp]: removed `/usr/lib/llvm-10/include`

File: package.json (hunk 4 above)
Comment (dm4) [reply package.json]: fixed

**Review by hydai — APPROVED** [review]

**Pull request merged**